.github/workflows/ci.yml

```diff
@@ -32,6 +32,10 @@ jobs:
           container: { image: "ubuntu:16.04", options: --privileged }
           qt: 5
         - name: Qt 5 / macOS x86_64
+          os: macos-12
+          container:
+          qt: 5
+        - name: Qt 5 / macOS arm64
```

Comment:
wow, there is an arm64 Qt5 sdk. Is it from homebrew?

Reply:
Yes it's still fetched and installed through homebrew